### PR TITLE
Add range filter for monthly_income field in matrimony profile admin

### PR DESCRIPTION
**Pull request by bhawanabadlani**
Added Numeric Range Filter for monthly_income. This filter helps in displaying the dataset within the given lower and upper range values of monthly_income.

### CODE REVIEW

File: vmb/matrimony/admin.py [hunk 1]
```diff
@@ -12,20 +15,76 @@ class MatrimonyProfileInline(admin.TabularInline):
     verbose_name = 'Language'
     verbose_name_plural = 'Languages Known'
 
-class MatrimonyProfileAdmin(admin.ModelAdmin):
+
+# class AgeFilter(admin.SimpleListFilter):
```
Comment (rtnpro) [inline vmb/matrimony/admin.py]: It's a good practice not to have unnecessary changes in a pull request. Changes that do not belong to this context. You can always backup these changes in a `tmp` branch and then copy things from there.

File: vmb/matrimony/models.py [hunk 2]
```diff
@@ -141,9 +141,9 @@ class MatrimonyProfile(models.Model):
     #Admins
     user = models.ForeignKey(User, on_delete=models.SET_NULL, null=True, blank=True)
 
-    def age(self):
-        if self.dob:
-            return int((datetime.datetime.now().date() - self.dob).days / 365.25)
+    # def age(self):
```
Comment (rtnpro) [inline vmb/matrimony/models.py]: same here

**Review by rtnpro — CHANGES_REQUESTED** [review]

File: vmb/matrimony/admin.py (hunk 1 above)
Comment (bhawanabadlani) [reply vmb/matrimony/admin.py]: okay...thank you

**Comment by rtnpro** [comment]
@bhawanabadlani Update the PR title here

image

to something meaningful and contextual.

**Review by rtnpro — APPROVED** [review]
